Examples/CodePushDemoApp/CodePushDemoAppTests/CheckForUpdateTests/testcases/NewUpdateTest.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+import React from "react-native";
+import CodePush from "react-native-code-push";
+let NativeCodePush = React.NativeModules.CodePush;
+import createTestCaseComponent from "../../utils/createTestCaseComponent";
+let PackageMixins = require("react-native-code-push/package-mixins.js")(NativeCodePush);
+import assert from "assert";
+import createMockAcquisitionSdk from "../../utils/mockAcquisitionSdk";
+
+let serverPackage = {
+  appVersion: "1.5.0",
+  description: "Angry flappy birds",
+  downloadUrl: "http://www.windowsazure.com/blobs/awperoiuqpweru",
+  isAvailable: true,
+  isMandatory: false,
+  packageHash: "hash240",
+  packageSize: 1024,
+  updateAppVersion: false
+};
+
+let localPackage = {
+  downloadURL: "http://www.windowsazure.com/blobs/awperoiuqpweru",
+  description: "Angry flappy birds",
+  appVersion: "1.5.0",
+  label: "2.4.0",
+  isMandatory: false,
+  isAvailable: true,
+  updateAppVersion: false,
+  packageHash: "hash123",
+  packageSize: 1024
+};
```

Comment:
not super important, but may be consider moving the above two(and other related redundant resources in these tests) packages to a resources.json(or something like that) file and read it once before running tests and assign them in here, so that in future if you have a change to the structure of acquisition package you don't have to make the change at multiple places. 

Maintenance of tests becomes a pain after a certain point, its best to keep them as low maintenance as possible.

Again, not super important, if you think it is worth it.